src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,10 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        boolean userIsNotNull = UserService.findByEmail(email) != null;
+        if (userIsNotNull && UserService.findByEmail(email).getPassword().equals(password)) {
+            return true;
+        }
         return false;
```

Comment:
For not using static you can create object of UserService

Reply:
yeah, thx. I wish I could see your message later.. I've done it